src/frontend/vite.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import tailwindcss from "@tailwindcss/vite";
+
+export default defineConfig({
+    plugins: [react(), tailwindcss()],
+    server: {
+        watch: {
+            usePolling: true,
+        },
+        host: true,
+        strictPort: true,
+        port: 9966,
+    },
```

Comment:
~~We had to add a [django proxy config as part of project template](https://github.com/azavea/azavea-project-template/pull/89/commits/b55e0b786aaf6bc0d1282579ce9bcc3b17c72d0f), do you think we need to add something similar here too?~~ Nevermind, I tried out `https://stg.echosearch.org/api/neighborhoods/` with the new deployment which is successful so I don't think the project template proxy config is necessary 🎉